apps/report/views.py

```diff
@@ -58,6 +61,7 @@ def get_context_data(self, **kwargs):
             "group__name").annotate(dcount=Count("group")).order_by("-dcount")
         context["repotest"] = Test.objects.values("git__name").annotate(
             dcount=Count("git")).order_by("-dcount")
+        context["testlengths"] = Task.objects.filter(datestart__gte = datetime.now()+timedelta(hours=-168)).values("test__name").annotate(avg_duration=Avg('duration')).order_by("-avg_duration")
```

Comment:
Fixed pep8 warnings in my newly added code. There are 2 more warnings in code unrelated to this PR.